src/video/resources/uploads.js

```diff
@@ -0,0 +1,94 @@
+/*!
+ * Mux Assets
+ * Copyright(c) 2018 Mux Inc.
+ */
+const Base = require('../../base');
+
+/**
+ * @private Base asset path for the Mux API
+ * */
+const PATH = '/video/v1/uploads';
+
+/**
+ * @private
+ * Build the base asset path for the Mux API
+ * */
+const buildBasePath = uploadId => `${PATH}/${uploadId}`;
+
+/**
+ * Uploads Class - Provides access to the Mux Video Uploads API
+ *
+ * @example
+ * const { Video } = new Mux(accessToken, secret);
+ *
+ * // Create an upload
```

Comment:
Shouldn't this be around line 38... or just removed?

Reply:
It's just an example for using the class module, but I was just being consistent with the other modules and don't feel too strongly about it.